capstone/src/main/java/com/google/sps/data/ReadData.java

```diff
@@ -7,9 +7,47 @@
 import java.io.BufferedReader;
 import java.io.FileReader;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
 
 public class ReadData {
   final String DELIMITER = ",";
+  static final Map<String,ArrayList> zipPrecinctMappint;
```

Comment:
Can we keep the Mapping in a separate class for readability? The ReadData class is only meant for reading the csv